sourcegraph/sourcegraph
# batches: fetch orgs when selecting a namespace

Closes #44708

![CleanShot 2022-12-02 at 16 19 09](https://user-images.githubusercontent.com/25608335/205326647-5d8a2e3d-fd5e-48ba-8627-7e38a9e1b9f0.gif)

## Test plan



Steps to reproduce:

* Create a new org
* Land on the org page
* Click on batch changes
* Click create
* The org namespace should be selected by default (previously it wasn't)

## App preview:

- [Web](https://sg-web-bo-fix-namespace-selector-new-org.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
